### DISCUSSION TITLE
Proton's 1.25px and 1.5px icons

### DISCUSSION THREAD

**Discussion by zapSNH** [black7375/Firefox-UI-Fix#845] [General]
Hi, I've noticed that some icons in the Firefox interface (particularly newer ones) have a slightly bigger stroke width (1.5px).
![image](https://github.com/black7375/Firefox-UI-Fix/assets/134786889/4cd313c5-7891-4246-aa19-90a717337f3e)
The middle icon has a 1.5px stroke width while the left and right icons have a 1.25px stroke

![image](https://github.com/black7375/Firefox-UI-Fix/assets/134786889/e34eb99b-6f7c-4c88-8645-5080ee29a6a7)
The Firefox view icon looks kinda blurry with a 1.5px stroke width

____

These could easily be replaced with 1.25px stroke width icons since there are so few of them.
However, [the Firefox design system site](https://acorn.firefox.com/latest/styles/iconography-q7JqGl5H) has 1.5px icons for everything including 1.5px versions of the old Proton icons. This might hint at a 1.5px version of Proton in the future.

IMO these 1.5px stroke icons are slightly worse than the 1.25px ones since they look blurry on a 100% DPI scale screen. But if an update comes that makes all of the icons have a 1.5px stroke width, will Lepton change these icons back or will Lepton embrace these icons?

**Comment by black7375** [comment]
I prefer `1.25px` icons.
So, I'm thinking about restoring it if possible.

However, if the number of mismatched icons increases dramatically and significantly, you may need to follow the `1.5px` icons. (It's not easy to replace them all)
